docs/dashboards/cortex-write.json

```diff
@@ -0,0 +1,870 @@
+               "legendFormat": "p95 - {{route}}",
+               "step": 30
+            },
+            {
+               "datasource": "default",
+               "expr": "histogram_quantile(0.99, sum(rate(cortex_request_duration_seconds_bucket{name=\"ingester\", route=~\".*Push\"}[1m])) by (le, route))",
+               "format": "time_series",
+               "intervalFactor": 2,
+               "legendFormat": "p99 - {{route}}",
+               "step": 30
+            }
+         ],
+         "thresholds": [ ],
+         "timeFrom": null,
+         "timeShift": null,
+         "title": "Latency",
+         "tooltip": {
+            "shared": true,
+            "sort": 0,
+            "value_type": "individual"
+         },
+         "type": "graph",
+         "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": [ ]
+         },
+         "yaxes": [
+            {
+               "format": "short",
+               "label": null,
+               "logBase": 1,
+               "max": null,
+               "min": null,
+               "show": true
+            },
+            {
+               "format": "short",
+               "label": null,
+               "logBase": 1,
+               "max": null,
+               "min": null,
+               "show": true
+            }
+         ]
+      },
+      {
+         "collapsed": false,
+         "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 8
+         },
+         "id": 5,
+         "panels": [ ],
+         "repeat": null,
+         "title": "Ruler",
+         "type": "row"
+      },
+      {
+         "aliasColors": { },
+         "bars": false,
+         "dashLength": 10,
+         "dashes": false,
+         "datasource": null,
+         "fill": 1,
+         "gridPos": {
+            "h": 7,
+            "w": 12,
+            "x": 0,
+            "y": 9
+         },
+         "id": 6,
+         "legend": {
+            "alignAsTable": false,
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "rightSide": false,
+            "show": true,
+            "total": false,
+            "values": false
+         },
+         "lines": true,
+         "linewidth": 1,
+         "nullPointMode": "null as zero",
+         "percentage": false,
+         "pointradius": 5,
+         "points": false,
+         "renderer": "flot",
+         "repeat": null,
+         "seriesOverrides": [ ],
+         "spaceLength": 10,
+         "span": 12,
+         "stack": false,
+         "steppedLine": false,
+         "targets": [
+            {
+               "datasource": "default",
+               "expr": "rate(cortex_rules_processed_total[1m])",
+               "format": "time_series",
+               "intervalFactor": 2,
+               "legendFormat": "{{ ft_container }}",
```

Comment:
This is pretty specific to FreshTracks, a different label would be better.

Reply:
Yes. Yes it would.